docs/library/ubluetooth.rst

```diff
@@ -181,6 +182,21 @@ Event Handling
             elif event == _IRQ_MTU_EXCHANGED:
                 # MTU exchange complete (either initiated by us or the remote device).
                 conn_handle, mtu = data
+            elif event == _IRQ_L2CAP_COC_ACCEPT:
+                # A new channel has been accepted.
+                chan_handle, = data
+            elif event == _IRQ_L2CAP_COC_CONNECT:
+                # A new channel is now connected (either as a result of connecting or accepting).
+                chan_handle, our_mtu, peer_mtu = data
+            elif event == _IRQ_L2CAP_COC_DISCONNECT:
+                # Existing channel has disconnected.
+                chan_handle, = data
+            elif event == _IRQ_L2CAP_COC_RECV:
+                # New data is available on the channel. Use l2cap_coc_recvinto to read.
+                chan_handle, = data
+            elif event == _IRQ_L2CAP_COC_TX_UNSTALLED:
```

Comment:
Should this be `SEND_UNSTALLED` to match the corresponding `l2cap_coc_send` method name?

Reply:
Yeah, agree. I've renamed this to `SEND_READY` just to make it less "double-negative", and changed `l2cap_send` to return True when "still ready" and "False" when "wait for the send ready event".